File: Maths/Statistics/Compound_Interest/Compound_Interest.c
#include <stdio.h>
#include <math.h>

int main(){
	double principle, rate, timeSpan;

    printf("Enter the principle amount\n");
    scanf(" %lf", &principle);

    printf("Enter the compound rate of interest\n");
    scanf(" %lf", &rate);

    printf("Enter the time span or number of times the amount gets compounded\n");
    scanf(" %lf", &timeSpan);
  
    /* Calculate compound interest */
    /*Formula to calculate final amount after applying compound interest is given by:
	FinalAmount = (principle)*((1 + (rate)/100)^(timeSpan))*/
    double FinalAmount = principle * (pow((1 + rate / 100), timeSpan)); 

    printf("Final amount after applying Compound interest = %lf\n", FinalAmount);
  
    return 0;
}

File: Maths/Statistics/Compound_Interest/Compound_Interest.java
//Calculating Compound Interest

import java.util.*;

class Solution
{
//function to calculate compound interest
    double compound_interest(double principle,double rate,double time){
	    return principle * Math.pow((1 + rate/100), time);
	}
}
public class Compound_Interest
{
//main class
    public static void main(String[] args)
    {
      	Scanner scanner = new Scanner(System.in);
	      Solution object = new Solution();
        double principle,rate,time;
        System.out.println("Enter the principle ");
      	principle = scanner.nextFloat();
      	System.out.println("Enter the rate ");
      	rate = scanner.nextFloat();
      	System.out.println("Enter the time period ");
	      time = scanner.nextFloat();
      	System.out.println("The compound interest is: "+object.compound_interest(principle, rate, time));
    }
} 
